css/main.css

```diff
@@ -325,3 +325,69 @@ li {
 .skills {
   list-style: inside url("../media/svg/circle.svg");
 }
+
+@media screen and (max-width: 700px) {
+  .formtext {
+    justify-content: center;
+    display: flex;
+    text-align: center;
+    color: #172b4d;
+    font-weight: 700;
+    font-size: 32px;
+    line-height: 30px;
+    padding: 1.5rem;
+  }
+
+  /* .firsttext {
+  } */
+  .forml .namepart {
+    background-color: #fcfcfc;
+  }
+
+  .btnn {
+    background: #ff6b00;
+    color: white;
+    padding: 12px;
+    margin-left: 8rem;
+  }
+
+  .bttn span::after {
+    content: "\00bb";
+    position: absolute;
+    opacity: 0;
+    top: 0;
+    right: -20px;
+    transition: 0.5s;
+  }
+
+  .bttn:hover span {
+    padding-right: 25px;
+  }
+
+  .bttn:hover span::after {
+    opacity: 1;
+    right: 0;
+  }
+
+  .textside {
+    border: 1px solid red;
```

Comment:
[OPTIONAL]

You may want to change the border color to something that looks closer to the Figma template :monocle_face: